language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/ServiceDeclarationTest.java

```diff
@@ -36,7 +36,7 @@
 
     @Override
     public List<String> skipList() {
-        return Collections.singletonList("config13.json");
+        return Lists.of("config13.json", "config17.json", "config18.json");
```

Comment:
Do we know why the langserver tests started failing? Shall we create an issue for these too?

Looping in @nadeeshaan too.